src/SfxWeb/src/app/json-typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.json" {
+    const value: any;
```

Comment:
this needs to be typed if you want to use a .json file to let intellisense properly handle the file for interpretting it.